spec/index.html

```diff
@@ -3684,8 +3684,10 @@ <h3>ORDER BY</h3>
           <li>Blank nodes</li>
           <li>IRIs</li>
           <li>RDF literals</li>
+          <li>Triple terms</li>
         </ol>
-        <p>SPARQL does not define a total ordering of all possible RDF terms. Here are a few examples
+        <p>SPARQL does not define a total ordering of all possible RDF terms. Implementations may
+          define total ordering through <a href="#operatorExtensibility">17.3.1 Operator Extensibility</a>. Here are a few examples
```

Comment:
Section numbers shouldn't be hard coded in the text as they may change. Either simply remove the number here:

```suggestion
          define total ordering through <a href="#operatorExtensibility">operator extensibility</a>. Here are a few examples
```
... or there is a ReSpec macro that you can use (I don't have it handy right now but you can find it being used in other parts of the doc).